mage_ai/frontend/components/PipelineDetail/AddNewBlocks/utils.tsx

```diff
@@ -51,6 +52,7 @@ const getDataSourceTypes = (
         DataSourceTypeEnum.INFLUXDB,
         DataSourceTypeEnum.S3,
         DataSourceTypeEnum.KAFKA,
+        DataSourceTypeEnum.NATS,
```

Comment:
You have not added the support for data exporter. Hence, this needs to be removed.